### ISSUE TITLE
Support loading Starlark templates from a branch

### ISSUE THREAD

**Issue by fkorotkov** [cirruslabs/cirrus-cli#366]
Right now `load` supports only specifying a `SHA` after `@`. It will be handy to allow specifying a branch name as well.

**Comment by edigaryev** [comment]
It's already possible, see [Remote from Git](https://github.com/cirruslabs/cirrus-cli/blob/master/STARLARK.md#remote-from-git) section of the CLI's Starlark documentation.

**Comment by fkorotkov** [comment]
Hm. Didn't work for me. Will try again in a bit. 👌

**Comment by fkorotkov** [comment]
Seems it doesn't work for

```python
load("github.com/cirrus-templates/graphql@initial-queries", "failed_instruction", "rerun_task")
```

I'm getting:

```console
cirrus run lint
Error: load failed: cannot load github.com/cirrus-templates/graphql@initial-queries: failed to retrieve a file from Git repository: reference not found
2021/04/14 18:53:51 load failed: cannot load github.com/cirrus-templates/graphql@initial-queries: failed to retrieve a file from Git repository: reference not found
```

**Comment by edigaryev** [comment]
Indeed, it only works for the default branch now.

Thanks for the test-case, should be fixed as a part of #367.

It would also be nice if we can keep this branch for the test to keep working.

**Comment by edigaryev** [comment]
The fix is available in the recently released [0.48.0](https://github.com/cirruslabs/cirrus-cli/releases/tag/v0.48.0).
